modules/ossm-cr-istio-global-1x.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/customizing-installation-ossm.adoc
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+[id="ossm-cr-istio-global_{context}"]
+= Istio global example
```

Comment:
These settings are now (mostly?) under `.spec.runtime.defaults`, so this entire document needs to be reorganized/rewritten.

Reply:
This file goes to the 1.x stuff, so if that hasn't changed, then it might be fine.